omod/src/main/webapp/resources/scripts/visit/vaccinations.js

```diff
@@ -9,8 +9,8 @@ angular.module("vaccinations", [ "constants", "ngDialog", "obsService", "encount
 
     })
 
-    .factory("VaccinationService", [ "Obs", "Concepts", "EncounterTypes", "EncounterTransaction", "EncounterRoles", "SessionInfo",
-        function(Obs, Concepts, EncounterTypes, EncounterTransaction, EncounterRoles, SessionInfo) {
```

Comment:
I take it turns out we were including these libraries but no longer using them?

Reply:
Correct